### PR TITLE
Add all relationship to node models and change jurisdiction to jurisd…

### PR DESCRIPTION
**Pull request by mariamera**

### CODE REVIEW

**Review by junior1q94 — APPROVED** [review]
LGTM

File: paradise_papers_search/fetch_api/models/helpers.py [hunk 1]
```diff
@@ -93,9 +94,9 @@ def serialized_realtionships_of_type(self, node_type):
         nodes   = []
 
         for row in results[0]:
-            node = self.nodes.get(node_id=row[1])
+            node = MODEL_ENTITIES[node_type].nodes.get(node_id=row[1])
```
Comment (oscarpolanco) [inline paradise_papers_search/fetch_api/models/helpers.py:97]: If you sent the entity to the function you can leave self here.
What you think? 🤔

File: paradise_papers_search/fetch_api/models/Officer.py [hunk 2]
```diff
@@ -31,17 +31,15 @@ def serialize(self):
...
                 'nodes_related': helpers.serialize_relationships(self.entities.all(), 'OFFICER_OF'),
             },
             {
-                'nodes_type': 'officer',
+                'nodes_type': 'Officer',
                 'nodes_related': helpers.serialized_realtionships_of_type(self, 'Officer'),
             },
         ]
-
-install_labels(Officer)
```
Comment (oscarpolanco) [inline paradise_papers_search/fetch_api/models/Officer.py:47]: 👍

File: paradise_papers_search/fetch_api/models/Address.py [hunk 3]
```diff
@@ -23,4 +23,3 @@ def serialize(self):
             },
         }
 
-install_labels(Address)
```
Comment (oscarpolanco) [inline paradise_papers_search/fetch_api/models/Address.py]: 👍

File: paradise_papers_search/fetch_api/models/helpers.py (hunk 1 above)
Comment (mariamera) [reply paradise_papers_search/fetch_api/models/helpers.py:97]: yes, but you can send something that is not the same node model. ex: 
on Intermediary.py there is 
` {
              'nodes_type': 'Officer',
            'nodes_related': helpers.serialized_realtionships_of_type(self, 'Officer'),
          },`

Link: https://github.com/neo4j-examples/paradise-papers-django/pull/38/files#diff-15361393f3dd0b79387fe15de18752f7R44

Comment (raulgomezc) [reply paradise_papers_search/fetch_api/models/helpers.py:97]: \+ 1, no need to index on the dictionary if we already have the type on self

**Review by raulgomezc — CHANGES_REQUESTED** [review]

**Review by raulgomezc — APPROVED** [review]

**Pull request merged**